deploy/template.yaml

```diff
@@ -1478,20 +1478,20 @@ Resources:
           - ProvisionedConcurrentExecutions: !FindInMap [ EnvironmentConfiguration, !Ref AWS::AccountId, provisionedConcurrency ]
           - !Ref AWS::NoValue
 
-  IPVProcessJourneyEventFunctionLogGroup:
+  IPVProcessJourneyStepFunctionLogGroup:
     Type: AWS::Logs::LogGroup
     Properties:
       RetentionInDays: 14
       LogGroupName: !Sub "/aws/lambda/process-journey-event-${Environment}"
```

Comment:
Should this update too?

Reply:
Would make sense to revert this now yeah, saves another PR later